v3 UI: [hotfix] post pages having background colours by default

bmndc opened the pull request:
- Fixed an issue where post background colour might be unintentionally applied without the corresponding option activated;
- Added a toggle to add or remove pictographs from the comment action buttons when Always Show Post Options is on;
- Keep up with Reddit changes.

joelacus commented:
I'm going to release version 2.1.0 tonight. Are there any other commits you want to do before then?

Also, there's a review on the [Google store page](https://chromewebstore.google.com/detail/reddit-enhancer/onglbklimdjicpdadjieknodkkmjldoa/reviews) about some UI elements not being rounded. I haven't noticed any issues with that, have you?

bmndc commented:
@joelacus 

> the comment buttons look very bad without icons

I've added a new toggle for people who have only been on Reddit since 2023 and need pictographs to read button text.

> UI elements (for example join buttton) [in the post sidebar] are squared instead of rounded

I just noticed this could happen on Chrome if "Hide post and sidebar dividers" is on when "Theme Corner Radius" hasn't been set. On Firefox, `border-radius` falls back to the previously set radius if the new value is invalid, but Chrome just assumes it's zero. Plus `addBorderRadiusToShadowRootElements()` runs and assigns `--re-theme-border-radius` to elements regardless whether "Theme Corner Radius" is activated or not.

I've added a failsafe for the function.

Reddit added a new page for tracking keywords ("Add Keywords" in the side menu), so I need to update RE to style that page. Should be done by the end of the day.
